cmd/manager/main.go

```diff
@@ -125,6 +127,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	TrackingStatus := strings.ToUpper(os.Getenv("TRACKING_STATUS"))
```

Comment:
Use variable `TRACKING_STATUS` to `ANALYTICS`
like:
```
isAnalytics := strings.ToUpper(os.Getenv("ANALYTICS"))
```

Reply:
Changes as requested